frontend/src/components/Header/StopClock.js

```diff
@@ -0,0 +1,101 @@
+// @flow
+import React, { Component } from 'react';
+import CircularProgressbar from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+
+type Props = {
+  startTime: number, // Date.now() ms at start
+  countdown: number, // delta in seconds
+  currentTime: number, // Date.now() at present
```

Comment:
You're not using this prop in the component.

Reply:
will remove